src/main/java/edu/hm/hafner/analysis/IssueDifference.java

```diff
@@ -31,6 +36,14 @@ public IssueDifference(final Report currentIssues, final String referenceId, fin
         fixedIssues = referenceIssues.copy();
         outstandingIssues = new Report();
 
+        referencesByHash = new HashMap();
```

Comment:
Please check the Java generics warnings:
https://github.com/jenkinsci/analysis-model/pull/553/checks?check_run_id=1662224597

Reply:
Done